src/System.Runtime.CompilerServices.Unsafe/ref/Configurations.props

```diff
@@ -3,6 +3,7 @@
     <BuildConfigurations>
       netstandard1.0;
       netstandard;
+      netcoreapp;
```

Comment:
@ericstj, is there anything else I need to do besides add this line in order to do this correctly?

Reply:
I don't think so. I think this should be enough. Should we instead make this a `.netstandard2.1` configuration?

`.netstandard2.1` already has nullable attributes defined.

Reply:
Changed to netstandard2.1